ui_src/src/components/getStartedItem/style.scss

```diff
@@ -21,11 +21,53 @@
         height: calc(100% - 120px);
         position: relative;
         padding-left: 40px;
+        .video-container {
+            display: flex;
+            justify-content: space-evenly;
+            padding-top: 10%;
+            padding-right: 40px;
+
+            .react-player__preview {
+                border-radius: 10px !important;
+                border: 1px solid #b4b4b4;
+                background-size: auto !important;
+                z-index: 1 !important;
+            }
+            .pink-ball {
+                top: 41.5%;
```

Comment:
When you use percentage, this breaks the responsiveness because not all the component is implemented with percentages.